### ISSUE TITLE
Unable to use gcr.io as a registry

### ISSUE THREAD

**Issue by remarkov** [cirruslabs/tart#166]
I'm on tart 0.16.0 and I'm trying to use gcr.io container registry as a registry for VMs. 

I have the credentials json data (same which I use for docker images push/pull so it has necessary access rights) in a file client-secret.json. I'm trying `cat client-secret.json | tart login gcr.io --username _json_key --password-stdin` but it immediately gives me an error `invalid credentials: UnexpectedHTTPStatusCode(when: "doing ping", code: 404, details: "")`

Then I tried with `https://gcr.io` and also with `gcr.io/{project_name}` or `https://gcr.io/{project_name}` - every time it just hangs indefinitely until I hit ^C a few times so it returns `invalid credentials: HTTPClientError.cancelled`.

I believe gcr.io must be OCI-compliant but I'm not 100% sure. 

Am I missing something? Did anyone else had any luck with tart + gcr.io? Any help on this issue would be much appreciated.

**Comment by edigaryev** [comment]
Hello Vladimir,

While Tart should work just fine with the [Artifact Registry](https://cloud.google.com/artifact-registry), there are indeed some issues with the older [Container Registry](https://cloud.google.com/container-registry), they will be addressed by https://github.com/cirruslabs/tart/pull/169.

Thanks for reporting!

**Comment by remarkov** [comment]
Hey @edigaryev,

Thanks a lot for such a quick fix! It seems the URL is correct now but something is still off with the credentials passing, here's what I got:

```
$ cat client-secret.json | tart login gcr.io --username _json_key --password-stdin

invalid credentials: AuthFailed(why: "received unexpected HTTP status code 400 while retrieving an authentication token", details: "{\"errors\":[{\"code\":\"UNKNOWN\",\"message\":\"Unable to parse json key.\"}]}")
```

The `client-secret.json` is a valid JSON file with Google credentials so seems like it's structure is being broken at some point.

**Comment by edigaryev** [comment]
> The `client-secret.json` is a valid JSON file with Google credentials so seems like it's structure is being broken at some point.

This should be now fixed as a part of [0.17.1 release](https://github.com/cirruslabs/tart/releases/tag/0.17.1).
